### PR TITLE
Add support for Quectel BG95 (SSL/TLS/HTTPS/GPS/GNSS)

### PR DESCRIPTION
**Pull request by aurelihein**
This commit add support for Quectel BG95 (SSL/TLS/HTTPS/GPS/GNSS)
I have also updated the library version number
Let me know if you need any modification
Thank you
Aurelien BOUIN

Note : Tested on ESP32S3 + BG95-M3

### CODE REVIEW

**Comment by aurelihein** [comment]
No review ?

File: src/TinyGsmClient.h [hunk 1]
```diff
@@ -77,6 +77,12 @@ typedef TinyGsmM95::GsmClientM95 TinyGsmClient;
 #include "TinyGsmClientBG96.h"
 typedef TinyGsmBG96                TinyGsm;
 typedef TinyGsmBG96::GsmClientBG96 TinyGsmClient;
+typedef TinyGsmBG96::GsmClientSecureBG96 TinyGsmClientSecure;
+
+#elif defined(TINY_GSM_MODEM_SECURE_BG95)
+#include "TinyGsmClientSecureBG95.h"
+typedef TinyGsmBG95                TinyGsm;
+typedef TinyGsmBG95::GsmClientSecureBG95 TinyGsmClientSecure;
```
Comment (shanedoolane) [inline src/TinyGsmClient.h:85]: This PR is for BG96 but why are you adding references to BG95?

File: README.md [hunk 2]
```diff
@@ -110,6 +111,7 @@ Watch this repo for new updates! And of course, contributions are welcome ;)
         - ESP8266 - 5
         - Neoway M590 - 2
         - Quectel BG96 - 12
+        - Quectel BG95 - 12
```
Comment (shanedoolane) [inline README.md:114]: BG95 or BG96?

File: README.md [hunk 3]
```diff
@@ -158,7 +160,7 @@ Watch this repo for new updates! And of course, contributions are welcome ;)
 
 **Location**
 - GPS/GNSS
-    - SIM808, SIM7000, SIM7500/7600/7800, BG96, u-blox
+    - SIM808, SIM7000, SIM7500/7600/7800, BG96, BG95, u-blox
```
Comment (shanedoolane) [inline README.md:163]: Same Question

**Comment by aurelihein** [comment]
I just updated the PR title

**Comment by happytm** [comment]
@aurelihein  I have tracker board HGD4-1.1 from System Loco. It has BG95-M3 modem , Nordic BLE chip as main controller and ESP8266 as web interface. Is it possible to use your code on Nordic chip?

Thanks.

**Pull request merged**